jib-gradle-plugin/CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ All notable changes to this project will be documented in this file.
 
 - Added support for [configuring registry mirrors](https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#i-am-hitting-docker-hub-rate-limits-how-can-i-configure-registry-mirrors) for base images. This is useful when hitting [Docker Hub rate limits](https://www.docker.com/increase-rate-limits). Only public mirrors (such as `mirror.gcr.io`) are supported. ([#3011](https://github.com/GoogleContainerTools/jib/issues/3011))
 
+### Changed
+
+- Build will fail if Jib cannot create or read the [global Jib configuration file](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#global-jib-configuration). ([#2996](https://github.com/GoogleContainerTools/jib/pull/2996))
```

Comment:
Maybe we should add this note to the cli changelog as well? Since we recently started supporting registry mirrors through the global Jib configuration file.

Reply:
I think that's okay. This doesn't apply to Jib CLI in some sense, because it's just a new feature addition for Jib CLI. Jib CLI didn't use the global jib config file before.